projects/ccf-rui/src/app/core/store/model/model.state.ts

```diff
@@ -163,12 +163,8 @@ export class ModelState extends NgxsImmutableDataRepository<ModelStateModel> {
     super.ngxsOnInit();
 
     this.referenceData = this.injector.get(ReferenceDataState);
-  }
-
-  ngxsAfterBootstrap(): void {
-    super.ngxsAfterBootstrap();
 
-    if (this.globalConfig.organ) {
+    if (this.globalConfig.organ && this.globalConfig.embedded) {
```

Comment:
Why is this needed? Why should this only be in embedded mode?

Reply:
The default organ should only show up in embedded mode. Otherwise the user selects the starting organ from the registration modal.

Reply:
It's not the default organ. When specified as such it is _the_ organ to load. Why does that only have to happen in embedded mode? Guess we need to revisit what embedded is used for. Is this change key for fixing the organ loading bug?

Reply:
Okay I guess I misunderstood the problem. The organs will load without this change but it only works when I take out baseHref. Let's talk about it tomorrow.